### DISCUSSION TITLE
连接vnc登录ec提示未联网

### DISCUSSION THREAD

**Discussion by Morningoh** [docker-easyconnect/docker-easyconnect#56] [Q&A] [answered]
我远程到vnc后登录vpn提示检查网络连接

<img width="577" alt="屏幕快照 2021-05-30 下午1 54 27" src="https://user-images.githubusercontent.com/33406533/120093732-a97bba00-c14e-11eb-93a8-ec7ba798030d.png">

**Comment by Hagb** [comment, marked as answer]
非默认端口，地址前需加协议名（`https://vpn.shanghaiwaterlaw.cn:4433`）
此外，这个服务端应该是很旧的那种了，应该是无法使用这些 EasyConnect 客户端来登陆的。

**Reply by Morningoh** [reply]
在哪里的地址前加这个协议名，终端输入的命令那里吗？麻烦告知下详细位置，十分感谢！
![屏幕截图 2021-05-31 163440](https://user-images.githubusercontent.com/33406533/120282474-b8986e80-c2ec-11eb-8b04-16684799a385.png)

**Reply by Hagb** [reply]
> 我远程到vnc后登录vpn提示检查网络连接
> 
> <img alt="屏幕快照 2021-05-30 下午1 54 27" width="577" src="https://user-images.githubusercontent.com/33406533/120093732-a97bba00-c14e-11eb-93a8-ec7ba798030d.png">

需要在这个界面上填写 `https://vpn.shanghaiwaterlaw.cn:4433`。

但我这里测试这个地址，会提示版本不匹配。登上这个地址，没有客户端下载按钮，只有控件下载（估计是老版 EasyConnect，但 docker-easyconnect 仅支持近年的独立 EasyConnect 客户端）

**Reply by Morningoh** [reply]
噢明白了，好的，谢谢，我再找找其他办法
